### PR TITLE
feat: add eth_getBlockTransactionCountByNumber

### PR DESCRIPTION
**Pull request by owenvoke**

### CODE REVIEW

File: src/Namespaces/Eth.php [hunk 1]
```diff
@@ -67,4 +68,20 @@ public function getBalance(string $address, string $defaultBlock = null): Wei
 
         return HexToWei::format($result);
     }
+
+    /**
+     * Returns the number of transactions in a block by its hash.
+     *
+     * @throws ErrorException|TransporterException
+     */
+    public function getBlockTransactionCountByNumber(string $blockHash): string
```
Comment (nunomaduro) [inline src/Namespaces/Eth.php]: Can you check if we don't accept a number, and we need to translate to hash before sending to the blockchain?

**Review by nunomaduro — CHANGES_REQUESTED** [review]
Its missing changes in our docs.

**Comment by nunomaduro** [comment]
@owenvoke Should I close this?

**Comment by owenvoke** [comment]
@nunomaduro, apologies, I'll rebase and apply the required changes for this. 👍🏻

**Pull request merged**